sros/docker/launch.py

```diff
@@ -354,9 +354,15 @@ def __init__(self, username, password, num_nics, mode):
         self.logger.info("Number of NICS: " + str(num_nics))
         self.logger.info("Mode: " + str(mode))
         # if we have more than 5 NICs or version is 19 or higher we use distributed VSR-SIM
-        if num_nics > 5 or major_release >= 19:
+        max_non_licensed_nics = 5
+        min_non_licensed_major_release = 19
+        if num_nics > max_non_licensed_nics or major_release >= min_non_licensed_major_release:
             if not self.license:
-                self.logger.error("More than 5 NICs require distributed VSR which requires a license but no license is found")
+                self.logger.error("More than {} ({} configured) NICs or a major version greater than {} ({} detected)".format(max_non_licensed_nics,
+                                                                                                                        num_nics,
+                                                                                                                        min_non_licensed_major_release,
+                                                                                                                        major_release) +\
+                                  " require distributed VSR which requires a license but no license is found")
```

Comment:
The recent base image change also upgraded Python to 3.9.2 so we can use f-strings:

```suggestion
                self.logger.error(f"More than {max_non_licensed_nics} ({num_nics} configured) NICs or a major version greater than {min_non_licensed_major_release} ({major_release} detected) require distributed VSR which requires a license, but no license is found")
```